login-workflow/src/screens/SuccessScreen/SuccessScreenBase.tsx

```diff
@@ -39,7 +40,28 @@ export const SuccessScreenBase: React.FC<SuccessScreenProps> = (props) => {
             <WorkflowCardHeader {...headerProps} />
             {Object.keys(instructionsProps).length !== 0 && <WorkflowCardInstructions {...instructionsProps} />}
             <WorkflowCardBody>
-                <WorkflowFinishState icon={icon} title={messageTitle} description={message} />
+                <Box
+                    sx={[
+                        {
+                            display: 'flex',
+                            flex: '1 1 0%',
+                            justifyContent: 'center',
+                            alignItems: 'center',
+                            height: '100%',
+                            minHeight: { md: 500, sm: 'auto' },
+                        },
+                    ]}
+                >
+                    {EmptyStateProps && (
+                        <EmptyState
+                            {...EmptyStateProps}
+                            sx={{
```

Comment:
The`EmptyStateProps` prop should be passed after `sx` prop, so that it should be overridden